graphqlTestExample.txt

```diff
@@ -0,0 +1,11 @@
+updateTest(id:"63fc1b3774319e02fbe49775" test:{
```

Comment:
what is this for?

Reply:
if you did some testing, feel free to just add a screenshot to your PR / attach code there! you don't need to commit it in the code

Reply:
oops uh I meant to delete this after but I forgot